server/src/leihs/procurement/routes.clj

```diff
@@ -6,11 +6,11 @@
     [clojure.string :refer [starts-with? replace]]
     [clojure.tools.logging :as log]
     [leihs.core.anti-csrf.back :as anti-csrf]
-    [leihs.core.auth.session :as session]
+    [leihs.core.auth.session2 :as session]
     [leihs.core.db :as db]
     [leihs.core.graphql :as core-graphql]
     [leihs.core.http-cache-buster2 :as cache-buster :refer [wrap-resource]]
-    [leihs.core.locale :as locale]
+    [leihs.core.locale2 :as locale]
```

Comment:
FYI: needed changes in shared-clj
https://github.com/leihs/leihs-clj-shared/pull/3/files

Reply:
FYI: removed all ***2 files again - seems to work now